@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -8,25 +8,20 @@ import {
     AttributeMap,
     composeTechniqueTextureName,
     DecodedTile,
-    getFeatureId,
-    getPropertyValue,
     ImageTexture,
     IndexedTechnique,
-    IndexedTechniqueParams,
     isLineMarkerTechnique,
     isPoiTechnique,
-    LineMarkerTechnique,
-    PoiGeometry,
-    PoiTechnique
+    MapEnv,
+    PoiGeometry
 } from "@here/harp-datasource-protocol";
-import { ContextualArabicConverter } from "@here/harp-text-canvas";
 import { assert, assertExists, LoggerManager } from "@here/harp-utils";
 import * as THREE from "three";
+import { Vector3 } from "three";
```

Comment:
THREE is already imported use THREE.Vector3

Reply:
done